### PR TITLE
feat(events-v2): Add filtering to heatmaps

### PR DESCRIPTION
**Pull request by lynnagara**
Filters heatmaps by project, environment, time and query

### CODE REVIEW

File: src/sentry/static/sentry/app/views/organizationEventsV2/tags.jsx [hunk 1]
```diff
@@ -28,19 +27,22 @@ class Tags extends React.Component {
...
     }
   }
 
   fetchData = async () => {
-    const {api, organization, view} = this.props;
+    const {api, organization, view, location} = this.props;
 
     try {
-      const tags = await fetchTags(api, organization.slug, view.tags);
+      const tags = await fetchTags(api, organization.slug, view.tags, location.query);
```
Comment (markstory) [inline src/sentry/static/sentry/app/views/organizationEventsV2/tags.jsx]: Does the fetchTags() action need to be updated to handle the datetime properties being structured differently?

File: src/sentry/static/sentry/app/views/organizationEventsV2/utils.jsx [hunk 2]
```diff
@@ -1,6 +1,7 @@
 import {pick} from 'lodash';
 
 import {DEFAULT_PER_PAGE} from 'app/constants';
+import {URL_PARAM} from 'app/components/organizations/globalSelectionHeader/constants';
```
Comment (billyvg) [inline src/sentry/static/sentry/app/views/organizationEventsV2/utils.jsx]: should this be moved out of this component specific dir?

File: src/sentry/static/sentry/app/views/organizationEventsV2/tags.jsx (hunk 1 above)
Comment (lynnagara) [reply src/sentry/static/sentry/app/views/organizationEventsV2/tags.jsx]: How are they structured differently?

File: src/sentry/static/sentry/app/views/organizationEventsV2/utils.jsx (hunk 2 above)
Comment (lynnagara) [reply src/sentry/static/sentry/app/views/organizationEventsV2/utils.jsx]: Moved!

File: src/sentry/static/sentry/app/views/organizationEventsV2/tags.jsx (hunk 1 above)
Comment (markstory) [reply src/sentry/static/sentry/app/views/organizationEventsV2/tags.jsx]: The data in the selection is nested inside a `datetime` property, whereas in the query string the  structure is flat.

**Review by markstory — APPROVED** [review]

File: src/sentry/static/sentry/app/views/organizationEventsV2/tags.jsx (hunk 1 above)
Comment (lynnagara) [reply src/sentry/static/sentry/app/views/organizationEventsV2/tags.jsx]: Ah, yeah it's just pulling it out of the browser's URL bar which is flat and not going through the stored structure

**Pull request merged**